src/__tests__/config-test.js

```diff
@@ -80,6 +80,42 @@ describe('Config urls', () => {
     {
       actual: urls.api.thingaggregate.scanOldUrl(1234, new MuseumId(99), new CollectionId('1234')),
       expected: '/api/thingaggregate/museum/99/scan?oldBarcode=1234&collectionIds=1234'
+    },
+    {
+      actual: urls.api.analysisType.getAllAnalysisTypes(new MuseumId(99)),
+      expected: 'api/management/99/analyses/types'
```

Comment:
The other paths starts with `/api/...` and not  `api/...`. I guess we expect the management paths to follow the same pattern